nilearn/decoding/searchlight.py

```diff
@@ -50,6 +50,9 @@ def search_light(X, y, estimator, A, scoring=None, cv=None, n_jobs=-1,
         adjacency matrix. Defines for each feature the neigbhoring features
         following a given structure of the data.
 
+    groups : array-like, optional
+        group membership for each sample for cross validation. default None
```

Comment:
I think we should write explicitly that groups is only available for scikit-learn version >= 0.18

May be we should add as a NOTE below default None.

NOTE: this argument is valid for scikit learn version >= 0.18 ?

Does this sounds good ? ping @GaelVaroquaux @bthirion